timescaledb/ubuntu/18.04/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2016-2019 Crave.io Inc. All rights reserved
+FROM accupara/ubuntu:18.04
+MAINTAINER Crave.io Inc. "contact@crave.io"
+
+RUN sudo apt-get update \
+ && sudo apt-get install -y \
+        bison \
+        build-essential \
+        flex \
+        gettext \
+        libgettextpo0 \
+        libkrb5-dev \
+        libossp-uuid-dev \
+        libpam0g-dev \
+        libperl-dev \
+        libreadline6-dev \
+        libssl-dev \
+        libtcl8.6 \
+        libxml2-dev \
+        libxslt1-dev \
+        perl-modules \
+        python \
+        python-dev \
+        python3-dev \
+        python3 \
+        readline-common \
+        tcl \
+        tcl-dev \
+        zlib1g-dev \
+        libxml2-utils \
+        xsltproc \
+ && sudo apt-get clean
```

Comment:
Please merge this with the next RUN.

Reply:
They have been kept separate on purpose:
First is for postgres and 2nd is for timescaledb.

Reply:
Oh I see.    
In that case, the apt-get clean is not useful is it?

Reply:
Just wanted to keep both the sections completely separate so that the layer that gets saved does not have the apt cache.